[Console][Legacy Editor] Ace Range implementation not being used

jloleysens opened the pull request:
## Summary

After merging this PR: https://github.com/elastic/kibana/pull/46793 a regression was introduced that affects the autocomplete system. This PR fixes it by using the `acequire` to load in the actual `ace` range implementation.

[Update]
## How to review

1. Open Console
2. type in: 
```
POST _search
{
   q[ctrl+space]
}
```
3. Ensure that the body of the request has correctly autocompleted.

elasticmachine commented:
Pinging @elastic/es-ui (Team:Elasticsearch UI)

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/7048/)
* Commit: 7f03bd93c3e04d9f5afa2f7e7e21c804cca87039

alisonelizabeth commented on src/legacy/core_plugins/console/np_ready/public/application/models/legacy_editor.test.ts:
```diff
@@ -0,0 +1,69 @@
+        end: { lineNumber: 2, column: 2 },
+      },
+      'test!'
+    );
```
should we add an `expect` statement here?

jloleysens replied:
This is intended as more a of smoke test, I don't actually intend to test output, more just does this actually execute a kind of thing. Will update the test title!

alisonelizabeth commented on src/legacy/core_plugins/console/np_ready/public/application/models/legacy_editor.test.ts:
```diff
@@ -0,0 +1,69 @@
+/*
```
Thanks for adding a test 👍

jloleysens commented:
@alisonelizabeth thanks for reviewing!

> ... more details on how to test the behavior that this PR is fixing

I updated the PR description with the missing steps for review.

>I added a comment regarding the test you added. I'm also getting ReferenceError: window is not defined when I try to run it.

Hmm, are you running it using:
```bash
yarn test:jest src/legacy/core_plugins/console/np_ready/public/application/models
```

I'm not seeing the `ReferenceError` log you mentioned.

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/7207/)
* Commit: 954c986a16104e4f85b6e88a7f582cf631bb88ef

jloleysens commented:
@elasticmachine merge upstream

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/7248/)
* Commit: 65c2715e6343757f5d892f36ec399560f54d80e9
